app/_components/systems/Form.tsx

```diff
@@ -0,0 +1,328 @@
+              variant="outline"
+              placeholder="Enter a minor faction..."
+              {...register('minorFaction')}
+            />
+            <FormErrorMessage>
+              {errors.minorFaction && errors.minorFaction.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%">
+          <FormControl
+            isInvalid={!!(errors.presenceType && errors.presenceType.message)}
+          >
+            <FormLabel>Presence Type</FormLabel>
+            <Select register={register('presenceType')}>
+              <option value="controlling">Controlling</option>
+              <option value="presence">Presence</option>
+            </Select>
+            <FormErrorMessage>
+              {errors.presenceType && errors.presenceType.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%">
+          <FormControl
+            isInvalid={!!(errors.stationFilter && errors.stationFilter.message)}
+          >
+            <FormLabel>Station Filter</FormLabel>
+            <Select register={register('stationFilter')}>
+              <option value="hasStations">Has Stations</option>
+              <option value="hasPlanetary">Has Planetary</option>
+              <option value="hasOrbital">Has Orbital</option>
+              <option value="hasNoStations">Has No Stations</option>
+            </Select>
+            <FormErrorMessage>
+              {errors.stationFilter && errors.stationFilter.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%">
+          <FormControl isInvalid={!!(errors.power && errors.power.message)}>
+            <FormLabel>Powers</FormLabel>
+            <PowersField register={register('power')} />
+            <FormErrorMessage>
+              {errors.power && errors.power.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%">
+          <FormControl
+            isInvalid={!!(errors.powerEffect && errors.powerEffect.message)}
+          >
+            <FormLabel>Power Effect</FormLabel>
+            <PowerEffectsField register={register('powerEffect')} />
+            <FormErrorMessage>
+              {errors.powerEffect && errors.powerEffect.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%">
+          <FormControl
+            isInvalid={!!(errors.factionState && errors.factionState.message)}
+          >
+            <FormLabel>Faction State</FormLabel>
+            <FactionStatesField register={register('factionState')} />
+            <FormErrorMessage>
+              {errors.factionState && errors.factionState.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%" colSpan={{ base: 1, md: 2, lg: 4 }}>
```

Comment:
I'm not actually sure what the second input was for, but I added it as it was on EDDB so it must have had a purpose - it could be that reference system specified to order the systems it finds by distances from the reference system? If so it was terrribly labelled on EDDB :joy: 

I'm going to leave it for the sake of EDDB form parity, unless you have an issue with that, as realistically it would be a very quick job to remove later if we don't need it, but a much longer job if we decide we do and have to re-add it. I guess we'll know more once the API is ready for system searches

![image](https://github.com/ed-pilots-network/frontend/assets/319883/87c19edf-e915-4110-9a08-9b27c1c06e29)

Reply:
Looking at the img, I think the top search might have been used as a fuzzy search to help the user recall a system name. Once they find what they are looking for they could use it in the form below as a reference for their search. So, as a UX question... is that necessary? If we have the reference system input in the beginning of the form and as the user types, the backend responds with a list of possibilities that is sufficient to me. Much like how Inara handles it.

Either way, the system search input will be rafactored here as well as the current /commodities page into a separate component that makes API calls to the backend to get valid system names so I'd keep it simple and drop the repeated input.

Reply:
Removed